docs/variants/asus_kgpe_d16/dasharo-compatibility/platform-suspend-and-resume.md

```diff
@@ -0,0 +1,44 @@
+# Dasharo compatibility: platform suspend and resume
```

Comment:
`compatibility` should start from a capital letter

Reply:
Improved